src/box/replication.cc

```diff
@@ -219,21 +222,44 @@ replication_init(int num_threads)
 	diag_create(&replicaset.applier.diag);
 
 	replication_threads = num_threads;
+	fiber_cond_create(&replication_connect_cond);
 
 	/* The local instance is always part of the quorum. */
 	replicaset.healthy_count = 1;
 
 	applier_init();
 }
 
+void
+replication_shutdown(void)
+{
+	replication_is_shutdown = true;
```

Comment:
`replication_is_shutting_down` would probably be a better name. Also, it'd be consistent with the names used in iproto.

Reply:
Done.